website/docs/reference/resource-configs/tags.md

```diff
@@ -16,59 +16,93 @@ datatype: string | [string]
 
 <File name='dbt_project.yml'>
 
+<VersionBlock lastVersion="1.8">
+
 ```yml
 
-models:
+[models](/reference/model-configs):
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
-snapshots:
+[snapshots](/reference/snapshot-configs):
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
-seeds:
+[seeds](/reference/seed-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+```
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+```yml
+
+[models](/reference/model-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[snapshots](/reference/snapshot-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[seeds](/reference/seed-configs):
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +tags: <string> | [<string>]
+
+[saved-queries:](/docs/build/saved-queries)
   [<resource-path>](/reference/resource-configs/resource-path):
     +tags: <string> | [<string>]
 
 ```
+</VersionBlock>
+
 
 </File>
 </TabItem>
 
 <TabItem value="other-yaml">
 
-<File name='models/resources.yml'>
+<VersionBlock firstVersion="1.9">
 
-```yml
-version: 2
+The following examples show how to add tags to dbt resources in YAML files. Replace `resource_type` with `models`, `snapshots`, `seeds`, or `saved_queries` as appropriate.
+</VersionBlock>
 
-models:
-  - name: model_name
-    config:
-      tags: <string> | [<string>]
```

Comment:
this change is wrong, at least for saved queries.  They support both string and [string] types for tags.